### DISCUSSION TITLE
Dynamic tag and move data between devices

### DISCUSSION THREAD

**Discussion by baxi10** [tughi/aggregator-android#79] [Ideas]
News reader as the very best. The ability to manually tag messages is a very good functionality. The great future could be to create tagged  messages based on search conditions. 
An important functionality is also the transfer of data between devices, which is also needed. 
--------
I found tag functionality based on search conditions very nice tool.

**Comment by tughi** [comment]
You can create tag rules that do just that... Have you looked into that?

**Reply by baxi10** [reply]
Yes, I looked it up, it's a great and useful option,
I discovered it after starting the discussion.

**Comment by tughi** [comment]
I'm currently working on a feature that exports all Aggregator's data to a backup file. This file can be then used to restore the state on a different device.

**Reply by baxi10** [reply]
Good. So I'm waiting for the work to be completed. Thanks
